src/dynarec/arm64/dynarec_arm64_00.c

```diff
@@ -3176,7 +3176,7 @@ uintptr_t dynarec64_00(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                     else
                         j64 = addr+i32;
                     jump_to_next(dyn, j64, 0, ninst, rex.is32bits);
-                    if(box64_dynarec_callret && addr >= (dyn->start + dyn->isize)) {
+                    if(box64_dynarec_callret) {
                         // jumps out of current dynablock...
```

Comment:
That's not what I meant.
I think the block can be changed to:
```C
                   MARK;
                   jump_to_next(dyn, j64, 0, ninst, rex.is32bits);
                    break;
```
Because the `if(...)` is basicaly `jump_to_next` with a fixed address.

Reply:
Oh, that will not work, the return point must be right after the `jump_to_next` (`PC + 4`), to be consistent with the last instruction of `jump_to_next`, which is `BLR x2`, which sets the `LR` to `PC + 4`.

Reply:
mmm, then I guess the if should be back to it's previous more complete test

Reply:
yeah, already force-pushed.